pepper-apis/src/main/java/org/broadinstitute/ddp/model/dsm/DsmNotificationEventType.java

```diff
@@ -33,4 +33,12 @@
      * Message sent by DSM when a result from a lab test on a kit is available
      */
     TEST_RESULT,
+    /**
+     * Message sent by DSM when a Circadia kit is delivered to participant
+     */
+    CIRCADIA_SENT,
+    /**
+     * Message sent by DSM when a Circadia  kit is received back
+     */
+    CIRCADIA_RECEIVED
```

Comment:
Can you point me at the ticket for this change? Does @SimoneMaiwald or @MocanaAtBroad know about these new notification types we're adding to the platform?

Reply:
I think it's this one:

https://broadinstitute.atlassian.net/browse/DDP-6574

Reply:
I'm not sure I like that we're adding notification types that's so study-specific (or at least, is named after the study). I know we did it as a special case for TestBoston, but still... What do you think @MocanaAtBroad?

Reply:
@yufengwng what if we renamed them as `KIT RECEIVED` and `KIT SENT`? it'll be more generic for the future events as well, @SimoneMaiwald what do you think?

Reply:
In the ticket, I see the original idea was to use `KIT_SENT` and `KIT_RECEIVED`. I'm curious why we decided to rename it to `CIRCADIA_*`? Also, how should we interpret the `KIT_*` types? How does it interact with other types like `BLOOD_SENT` and `SALIVA_RECEIVED`? Do we get both `KIT_SENT` and `BLOOD_SENT` when a blood kit is sent?

From what I can tell, these notification types are pretty isolated in DSS -- they don't really affect other features in the platform. So having something like `CIRCADIA_SENT` is fine in a sense. But I urge us to think a bit more about these since there's other studies we support and we'll need to document and explain why we have things like `CIRCADIA_SENT` in the platform.

Reply:
I was using `CIRCADIA_SENT` and `CIRCADIA_RECEIVED` because they were provided in the comments of the ticket, we can return those to the original values, so that they can be `KIT_SENT` and `KIT_RECEIVED`. I'm not sure about the reason of renaming to be honest, I was just following the requested solution.

The main reason of having these notifications is that they are used as `triggers` to create some activities (e.g in this case `DLMO Instructions` and `Kit Receipt Attestation`), also they are used in `emailing` too.

I don't know whether these notification types interact with other types in any way, for Circadia we just needed special triggers from DSM to control workflows and statuses.

I'm open to be included in more detailed discussion if it's necessary, maybe a special meeting with others too/

Reply:
Actually the real `trigger` is specified as this: `As soon as Kit Shipped Date is entered in DSM` and I'm note sure how it should happen in DSM since I'm not familiar with that project and also I'm not sure whether we have a similar `trigger` in DSS side.

We were using `KIT_SENT` and `KIT_RECEIVED` just to start out with that problem